src/types.ts

```diff
@@ -83,6 +83,8 @@ export interface FiatConnectClientConfig {
   baseUrl: string
   providerName: string
   iconUrl: string
+  celoNetwork: 'alfajores' | 'mainnet'
```

Comment:
ultra nit: we use alfajores and mainnet as keys elsewhere (for chain ids). might be nice to have a type for them